_docs/kustomize/akash-node/kustomization.yaml

```diff
@@ -35,7 +35,6 @@ configMapGenerator:
       - fast_sync=true
       - db_backend=goleveldb
       - db_dir=data
-      - log_level=main:info,state:info,*:error
```

Comment:
Don't we want to keep this as a configuration option? It still works if you set `info` from what I could tell

Reply:
we can keep it. `info` is a default level though